src/main/java/com/digitalasset/testing/utils/SandboxUtils.java

```diff
@@ -59,4 +64,17 @@ public static void waitForSandbox(DamlLedgerClient client, Duration waitTimeout,
     if (connected) logger.info("Connected to sandbox.");
     else throw new TimeoutException("Can't connect to sandbox");
   }
+
+  public static Predicate<Path> damlYamlP() {
+    return p -> Objects.equals("daml.yaml", p.getFileName().toString());
+  }
+
+  public static Path findDamlYaml(Path p) {
```

Comment:
I'd suggest to remove this logic completely, as it replicates a job that the `daml` assistant should do.

Reply:
In a normal project, the test library code will be running in the project root (although I can see the use case when we want to use this property though). Now, we are testing the library against a test project, which is under `src/main/resources/pingpong`, but the working directory for the test library during the test run is the project root of this whole project. Which is not what we want, we want the project root will be `src/main/resources/pingpong` in our tests.
Since I can see why someone want to use this option, I think we should keep it.

Now the behavior of the daml executable is to look for a daml.yaml in one of the parent folders, and if there is none, it will use the default SDK whatever that is, silently. Question is: do we want that behavior, or we want to error out. I tend to favor the latter, since that is more explicit. I just got bitten by this behavior... But this is something we need to decide on.

There is another option, to not rely on daml.yaml, but use and environment variable to specify the SDK version, but I think that's not a real option. We might want other settings from daml.yaml as well.

Now the implementation details of this is another question. We had to hurry with this PR, so a lot of quick'n dirty stuff went in. It would be very good to clean this whole mess up, with proper error handling, etc.